### PR TITLE
App styling

### PR DESCRIPTION
**Pull request by kennyegun24**
## In this PR, I styled the App to my taste

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ form.addEventListener('submit', (e) => {
```
Comment (HAMMAS-SALEEM) [inline src/index.js:10]: Missing-commands

- [x] In your setup section you have made it professional but i guess you forgot to add commands of how to clone your repository 😄 No problem you can add them now 😊

Comment (HAMMAS-SALEEM) [inline src/index.js:10]: License-link

- [x] Your link to your license file is broken. Kindly fix it 👍

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @kennyegun24 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
:heavy_check_mark: Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (kennyegun24) [reply src/index.js:10]: Thank you for notifying me on this, I have done it now

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @kennyegun24,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**